package.json

```diff
@@ -51,17 +59,32 @@
     "typedoc": "^0.17.2",
     "typescript": "^3.8.0"
   },
-  "lint-staged": {
-    "*.ts": [
-      "prettier --write",
-      "git add"
+  "eslintConfig": {
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+      "eslint:recommended",
+      "plugin:@typescript-eslint/eslint-recommended",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:jest/recommended",
+      "plugin:import/errors",
+      "plugin:import/warnings",
+      "plugin:import/typescript",
+      "prettier",
+      "prettier/@typescript-eslint"
     ]
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "*.ts": [
+      "eslint --fix",
+      "prettier --write",
```

Comment:
Question: Is it necessary to have ESLint and Prettier running given that `eslint-config-prettier` is set up now?

Reply:
I'm not positive but I _think_ explicit invocation of both is still useful as-is in this PR.

If we wanted to combine the steps we could use `eslint-plugin-prettier`

Reply:
Ah, makes sense to me, I got `eslint-plugin-prettier` and `eslint-config-prettier` mixed up in my head. Thank you!